### ISSUE TITLE
Quickstart

### ISSUE THREAD

**Issue by LaloHao** [stamourv/picobit#6]
Is there any tutorial on how to start playing with picobit on a microcontroller? 
A blinky led or something like that

I couldn't find any link/source code
Sorry if this seems too obvious, im EE student (learning lisp by my own) and i want to drop low level languages

**Comment by stamourv** [comment]
Not really. Getting it to work is very dependent on the microcontroller itself, and on the system and hardware around it.
If you do want to write something up for the specific system you're using, that would be great, though!

**Comment by LaloHao** [comment]
I'd like to do that but i have no idea where to start, i've mostly used C on top of Microchip's IDE, i'm guessing i need to get all register's addresses to configurate clock/digital ports/adc/etc and then implementate those registers from picolisp?
If this is already done for any particular PIC that'd be great (it says it works on 18F family on the project description), i wouldn't start from scratch, ultimately i want to port several PICs, of course all of this would be open source

**Comment by stamourv** [comment]
Your plan sounds reasonable.
I don't have the code handy for the board that we used (that was 6+ years ago), sorry.
